tests/integration/ha_tests/test_self_healing.py

```diff
@@ -145,8 +148,9 @@ async def test_storage_re_use(ops_test, continuous_writes):
 @pytest.mark.group(1)
 @pytest.mark.abort_on_fail
 @pytest.mark.parametrize("process", DB_PROCESSES)
-async def test_kill_db_process(
-    ops_test: OpsTest, process: str, continuous_writes, primary_start_timeout
+@pytest.mark.parametrize("signal", ["SIGTERM", pytest.param("SIGKILL", marks=markers.juju2)])
```

Comment:
We shouldn't skip test on Juju3

Reply:
This is currently being skipped on the K8s repo, should it get reverted over there too or there's a reason for it?

Reply:
IIRC there were issues on k8s specifically, we should check and reenable there whenever possible. But for VM the test should be working.

Reply:
Done, test pass as normal.